### ISSUE TITLE
Not able to display nodes and arrows by running "basic example" from /README.md file

### ISSUE THREAD

**Issue by gauravgrover95** [HelikarLab/ccNetViz#6]
OS: MacOS High Sierra 10.13.1
Browser: Firefox Quantum  58.0.2 (64-bit)

Following I have attached the screenshot of the result I obtained by running the "basic example" mentioned in the /README.md documentation file of the project.

Please note that no nodes and arrows are visible. As a result, there is no distinction between the source and the target nodes. 

In addition to it, I tried zooming to the very large level. The exclamation mark (!) was just being visible as a single straight vertical line. 

---

![ccnetviz screenshot](https://user-images.githubusercontent.com/13467940/36951200-d7d7dfae-2026-11e8-9d40-4811d18705ef.png)

---

**Comment by gauravgrover95** [comment]
I am trying to fix it but, As you can see in above screenshots, the library is not throwing errors. I am not getting any indication about what could be the exact problem?

I hypothesized that since both arrows and nodes are missing. There must be some problem in rendering the PNG files from the image folder. But When I am running examples from inside the /examples folder inside the library. They are just running fine.

Any Thoughts @saskaale ?

**Comment by gauravgrover95** [comment]
I figured out what the problem was. Following were the steps I performed.

Step 1: I made a new folder named "xyz-project". // the folder name really did not matter.
Step 2: I made a new file named index.html where I placed all the code in the "Basic Example".
Step 3: I cloned the ccNetViz repository.
Step 4: I modified the path name of the ccNetViz library in script tag. As, there was error for "file not found" for ccNetViz library in console.
Step 5: Shooted up a Python SimpleHttpServer and tested in browser. Where I was receiving the above mentioned results unexpected **without any error**.


The problem was that I did not correct the path for image files in _options.styles_ for _nodes_ while instantiating ccNetViz in the following code:


```

    var graph = new ccNetViz(document.getElementById('container'), {
      styles: {
        node: { texture: "images/node.png", label: { hideSize: 16 } }, // --> Here was the problem
        edge: { arrow: { texture: "images/arrow.png" } } // --> Here was the problem again
      }});
```


I think I can close this particular issue and file another issue that the Styles code is not throwing any error if the file was not found.
